topic_message_submit_transaction.go

```diff
@@ -127,6 +127,9 @@ func (transaction *TopicMessageSubmitTransaction) SignWith(
 }
 
 func (transaction *TopicMessageSubmitTransaction) validateNetworkOnIDs(client *Client) error {
+	if client == nil {
+		return nil
+	}
 	if !client.autoValidateChecksums {
```

Comment:
Can't these be combined for succinctness?
```suggestion
	if client == null || !client.autoValidateChecksums {
```

Reply:
First mistake is trying to be succinct in Go. :laughing: